packages/preview/realhats/0.1.0/README.md

```diff
@@ -0,0 +1,54 @@
+**realhats** puts real hats on symbols in math mode. Why settle for the lifeless, old-fashioned circumflex when your variables could sport berets, sombreros, top hats, and more? Inspiration and most hats from the original [realhats for LaTeX](https://github.com/mscroggs/realhats).
+
+Before and after realhats:
+
+<div style="display: flex; gap: 10px;">
+  <img src="example-1.png" alt="Without realhats" width="48%;">
+  <img src="example-2.png" alt="With realhats" width="48%;">
+</div>
+
+## Using realhats
+
+[Install](https://github.com/typst/packages?tab=readme-ov-file#local-packages) realhats into your local package directory under version 0.1.0.
```

Comment:
This should not be the recommend way of installing the package. Can you add a sentence above showing how to import via the `@preview` namespace, and present the `@local` installation as an alternative?